runtime/vm/ContinuationHelpers.cpp

```diff
@@ -292,6 +293,18 @@ walkContinuationStackFrames(J9VMThread *currentThread, J9VMContinuation *continu
 	return rc;
 }
 
+
+jvmtiIterationControl walkContinuationCallBack(J9VMThread *vmThread, J9MM_IterateObjectDescriptor *object, void *userData) {
+	J9VMContinuation *continuation = J9VMJDKINTERNALVMCONTINUATION_VMREF(vmThread, object->object);
+	if (NULL != continuation) {
+		/* Reset localWalkState */
+		J9StackWalkState localWalkState = *(J9StackWalkState*)userData;;
```

Comment:
We are not reusing localWalkState. So, the reset comment does not apply here. Also, the extra `;` should be removed.

Reply:
```suggestion
		J9StackWalkState localWalkState = *(J9StackWalkState*)userData;
```